docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### High Availability in PMM
+
+
```

Comment:
Let's put the main benefit here before HA explanation? After this paragraph, we can link with HA in PMM epic?

> If PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server comes back online, which is when metrics will get pushed to PMM Server. Note that this only applies to PMM Clients which run in the so-called push mode. The same holds true when a brief network outage occurs on the client side.

Reply:
Roma suggested a different description like this:

> When the connection between  DB and PMM is broken for a short period (what ? ) the data will be back when connection restored. So you  have visibility on behavior even when the network is broken."